src/Components/Popover/Base/PopoverDialog.tsx

```diff
@@ -46,6 +46,8 @@ export interface PopoverDialogProps extends StandardFunctionProps {
   enableDefaultStyles: boolean
   /** Allows the popover to dismiss itself when the trigger is no longer in view */
   onHide: () => void
+  /** This keeps the Popover visible no matter what */
+  visible?: boolean
```

Comment:
does `visible` default to `true` or `false`? if it defaults to false, it's probably fine to leave this an optional param. if it should default to `true`, we should probably make it required and provide a default boolean value of `true` on line 64 below

Reply:
It is false by default, but it wouldn’t hurt to have that more explicitly stated